jekyll/_cci2/concepts.md

```diff
@@ -8,33 +8,48 @@ order: 1
 ---
 
 
-A CircleCI project shares the name of the associated code repository and is visible on the Projects page of the CircleCI app. Projects are added by using the Add Project button.
+A CircleCI project shares the name of the associated code repository. Select Add Project from the CircleCI application to enter the Projects dashboard, from here you can set up or follow projects you have access to.
 
 * TOC 
 {:toc}
 
-A CircleCI project shares the name of the associated code repository and is visible on the Projects page of the CircleCI app. Projects are added by using the Add Project button.
+## Add Projects
 
 On the "Add Projects" page, you can either _Set Up_ any project that you are
 the owner of on your VCS, or, _Follow_ any project in your organization to gain
 access to its pipelines and to subscribe to [email notifications]({{
 site.baseurl }}/2.0/notifications/) for the project's status.
 
-## Add Projects Page
-
 {:.tab.addprojectpage.Cloud}
 ![header]({{ site.baseurl }}/assets/img/docs/CircleCI-2.0-setup-project-circle101_cloud.png)
 
 {:.tab.addprojectpage.Server}
 ![header]({{ site.baseurl }}/assets/img/docs/CircleCI-2.0-setup-project-circle101.png)
 
+## User Types
+
 The *Project Administrator* is the user who adds a GitHub or Bitbucket
 repository to CircleCI as a Project. A *User* is an individual user within an
 org. A CircleCI user is anyone who can log in to the CircleCI platform with a
 username and password. Users must be added to a [GitHub or Bitbucket org]({{
 site.baseurl }}/2.0/gh-bb-integration/) to view or follow associated CircleCI
 projects.  Users may not view project data that is stored in environment variables.
 
+## Config
+
+CircleCI believes in *configuration as code*. The entire pipeline process, from build to deploy, is orchestrated through a single file called `config.yml`.  The `config.yml` file is located in a folder called `.circleci` at the top of your project. CircleCI uses the YAML syntax for config, see the [Writing YAML]({{ site.baseurl }}/2.0/writing-yaml/) document for basics.
```

Comment:
I would recommend `root of your project` instead of `top`.

Reply:
Oh yeah good call 👍 :-)